src/holodeck/util.py

```diff
@@ -158,3 +158,17 @@ def draw_point(env, loc, color=None, thickness=10.0):
     color = [255, 0, 0] if color is None else color
     command_to_send = DebugDrawCommand(3, loc, [0, 0, 0], color, thickness)
     env._enqueue_command(command_to_send)
+
+
+def logs_path():
+    """Gets path for logs.
+
+    Returns:
+        :obj:`str`: The file path of where the logs are located
+    """
+    if os.name == "nt":
+        print(os.path.expanduser(r'~\AppData\Local\UnrealEngine\4.22\Saved\Logs'))
+        print(os.path.expanduser(r'~\AppData\Local\UnrealHeaderTool\Saved\Logs'))
+        print(os.path.expanduser(r'~\AppData\Local\UnrealBuildTool'))
```

Comment:
It looks like the logs for the Holodeck engine (for DefaultWorlds) itself are at ` ~\AppData\Local\holodeck\0.3.0\worlds\DefaultWorlds\WindowsNoEditor\Holodeck\Saved\Logs` on Windows. Both points in my comment [here](https://github.com/BYU-PCCL/holodeck/pull/439/files#r554481608) also apply here.